### ISSUE TITLE
Please add support for default UNIX "date" command format

### ISSUE THREAD

**Issue by dswhite42** [scrapinghub/dateparser#121]
Most Unix/Linux flavors use **`%a %b %e %T %Z %Y`** as the default date format.  However dateparser does not support that format.

    > date
    Tue Oct 13 20:18:56 CDT 2015
    
    > python
    >>> import dateparser
    >>> type(dateparser.parse('Tue Oct 13 20:18:56 CDT 2015'))
    <class 'NoneType'>

**Comment by asadurski** [comment]
Hello @dswhite42. Can you check your version of dateparser? The issue you mentioned was fixed in release 0.3. Just to prove it, here's the output from the current version:
```
In [1]: from dateparser import __version__

In [2]: print __version__
0.3.0

In [3]: from dateparser import parse

In [4]: parse('Tue Oct 13 20:18:56 CDT 2015')
Out[4]: datetime.datetime(2015, 10, 14, 6, 18, 56)
```

Please let me know if updating fixed your problem.

**Comment by dswhite42** [comment]
Thank, Artur, but that is indeed the version I have.  This is running under Python 3.4.3 (RHEL 6.2).  Here's a full session:

    > python3

    Python 3.4.3 (default, Jul  7 2015, 11:52:48)
    [GCC 4.4.7 20120313 (Red Hat 4.4.7-11)] on linux
    Type "help", "copyright", "credits" or "license" for more information.

    >>> import dateparser                           
    >>> print(dateparser.__version__)
    0.3.0
    >>> good = dateparser.parse('10/13/2015 20:18:56')
    >>> print(good)
    2015-10-13 20:18:56
    >>> type(good)
    <class 'datetime.datetime'>
    >>>
    >>> bad = dateparser.parse('Tue Oct 13 20:18:56 CDT 2015')
    >>> print(bad)
    None
    >>> type(bad)
    <class 'NoneType'>

**Comment by asadurski** [comment]
My bad. The fix was introduced after the 0.3 release, with PR #104. If you need it quickly, you can use the latest version of dateparser from master branch, or wait for next release, which comes in a couple of days.

**Comment by asadurski** [comment]
This problem has been fixed in [release 0.3.1](https://github.com/scrapinghub/dateparser/releases/tag/v0.3.1). Enjoy!

**Comment by dswhite42** [comment]
Confirmed!  Thanks very much.
